src/App.js

```diff
@@ -57,9 +56,20 @@ class App extends React.Component {
     });
   }
 
-  toggleCompleted = () => {
-    console.log("it works!")
-  }
+  toggleCompleted = (id) => {
+    this.setState({
```

Comment:
const target = this.state.data.find(data => data.id === id);
    console.log(target);
    target.completed = !target.completed; 
    this.setState({
      data: [...this.state.data, target]
    })
*Just an idea